### ISSUE TITLE
Missing snippet directory for MELPA install

### ISSUE THREAD

**Issue by kyleam** [stan-dev/stan-mode#39]
I get the following error on startup after installing stan-snippets from MELPA.

    File error: Opening directory, no such file or directory,
    /home/kyle/.emacs.d/.cask/24.4.1/elpa/stan-snippets-20141223.1420/snippets

I believe 3e053ab027c316fc7c67e18201166e6b777f1801 is to blame because the top-level 'snippets' directory specified in the MELPA recipe is no longer present.

Changing the MELPA recipe to use something like below should fix it (and make some of the symlinks in the root directory unnecessary).

```diff
diff --git c/recipes/stan-snippets w/recipes/stan-snippets
index 977d290..55887cf 100644
--- c/recipes/stan-snippets
+++ w/recipes/stan-snippets
@@ -1,4 +1,8 @@
 (stan-snippets
  :fetcher github
  :repo "stan-dev/stan-mode"
- :files ("stan-snippets.el" "snippets"))
+ :files ("stan-snippets/stan-snippets.el"
+         ("snippets/stan-mode"
+          "stan-snippets/snippets/stan-mode/.yas-compiled-snippets.el"
+          "stan-snippets/snippets/stan-mode/.yas-make-groups"
+          "stan-snippets/snippets/stan-mode/.yas-parents")))
```

**Comment by bob-carpenter** [comment]
Thanks for the patch!

- Bob

**Comment by jrnold** [comment]
Thanks! Sorry, I had refactored to repository to split up the various packages, but hadn't completed pull requests for melpa (hence the unnecessary symlinks to maintain backwards compatibility with the current melpa recipes).  I have some melpa pull requests open to resolve this: milkypostman/melpa#2703 .

**Comment by jrnold** [comment]
Merged into melpa in milkypostman/melpa@eda8539b7d8da3a458a38f7536ed03580f9088c3.

Fixed!

**Comment by kyleam** [comment]
The new MELPA recipe works for me.  Thanks!
